confuse/cache.py

```diff
@@ -0,0 +1,121 @@
+from typing import Dict, List
+
+from . import templates
+from .core import ROOT_NAME, Configuration, ConfigView, RootView, Subview
+
+
+class CachedHandle(object):
+    """Handle for a cached value computed by applying a template on the view.
+    """
+    # some sentinel objects
+    _INVALID = object()
+    _MISSING = object()
```

Comment:
Maybe some short comments on these to explain what they mean would be helpful?

Reply:
Added.